WallPaperDev/KabegamiTests/KabegamiTests.swift

```diff
@@ -0,0 +1,187 @@
+//
+//  KabegamiTests.swift
+//  KabegamiTests
+//
+//  Created by Jackson Ho on 3/30/20.
+//  Copyright © 2020 Stephen Ouyang. All rights reserved.
+//
+
+import XCTest
+
+@testable import Kabegami
+
+class KabegamiTests: XCTestCase {
+  
+  private let themeManager = ApplicationDependency.manager
+  
+  func testLocalizedString() {
```

Comment:
Nit - If the test scheme changes to a non English language these tests will fail.  Not sure if we need to see if they exactly match? Maybe just that the key does not equal the value? (meaning there is a string for that key)

Reply:
@alexander-dejeu  Hmm, that makes sense. Again, I'm not really familiar on what to test for Localized String. So i would love if you can elaborate on your suggestions.